src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import Greeting from './Greeting';
+import store from './redux/config/store';
+
+const App = () => (
+  <Provider store={store}>
+    <Greeting />
+  </Provider>
+);
+
+export default App;
```

Comment:
- [ ] Kindly add react-router to your React application and ensure that the `App.js` component has a route that leads to the Greeting component. Cheers 👍

Reply:
Alright, thanks very much for the review @sumairq , I just resolved it